Snowflake.Data.Tests/SFConnectionPoolT.cs

```diff
@@ -12,16 +12,47 @@ namespace Snowflake.Data.Tests
     using System.Threading.Tasks;
     using System.Threading;
     using Snowflake.Data.Log;
-    using System.Diagnostics;
     using Snowflake.Data.Tests.Mock;
-    using System.Runtime.InteropServices;
     using System.Data.Common;
+    using Moq;
 
-    [TestFixture]
+    
+    readonly struct PoolConfig {
```

Comment:
1. it can be a class
2. it can has parameterless constructor reading defaults from SnowflakeDbConnectionPool once before all tests
3. it can have method `setDefaultSnowflakeDbConnectionPool` which work on SnowflakeDbConnectionPool
4. then all the fields could be private